osquery/logger/plugins/tls_logger.cpp

```diff
@@ -58,6 +60,43 @@ Status TLSLoggerPlugin::logStatus(const std::vector<StatusLogLine>& log) {
   return forwarder_->logStatus(log);
 }
 
+Status TLSLoggerPlugin::logEvent(const std::string& s) {
+  std::string capture_event_types = Flag::getValue("logger_tls_event_types");
```

Comment:
Accessing FLAGS_logger_tls_event_types is thread unsafe, since logger_tls_event_types can be set from the configuration.